CHANGES/7154.bugfix

```diff
@@ -0,0 +1 @@
+Gave Pulp2ContentSerializer.pulp3_content its required view-name.
```

Comment:
How about "eliminated the view_name warning during the [re]start of Pulp services" or something along those lines?"
I'm trying to come up with something user-focused to keep it as .bugfix.

Reply:
Good point, fixed